### PR TITLE
test/add_fivetran_id

### PR DESCRIPTION
**Pull request by fivetran-catfritz**
**Are you a current Fivetran customer?** 

Fivetran created PR

**What change(s) does this PR introduce?** 

## 🔧 Bug Fixes
- Updated models `stg_pendo__feature_event` and `stg_pendo__page_event` to include `_fivetran_id`, which was recently added per the Fivetran Pendo Connector's [release notes](https://fivetran.com/docs/applications/pendo/changelog#december2022). ([#19](https://github.com/fivetran/dbt_pendo_source/pull/19))

## ✨ Features
- Updated documentation and packages to reference the latest version of the source package. ([#20](https://github.com/fivetran/dbt_pendo/pull/20))
- Revised readme instructions for successfully setting up the `GROUP` table with Snowflake. ([#20](https://github.com/fivetran/dbt_pendo/pull/20))

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [x] Yes (please provide breaking change details below.)
- [ ] No  (please provide an explanation as to how the change is non-breaking below.)
Added `_fivetran_id` in source.

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, Issue/Feature [#18](https://github.com/fivetran/dbt_pendo_source/issues/18)
- [ ] No 

**How did you test the PR changes?** 

 

- [x] Buildkite  
- [x] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [x] Postgres
- [x] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
♦️

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,17 +132,24 @@ vars:
...
-
 ```yml
 vars:
-  pendo_group_identifier: '"Group"' # as an example, must include this quoting pattern and your exact casing
+  pendo_group_identifier: '"Group"' # as an example, must include this quoting pattern and adjust for your exact casing
+```
 
-quoting:
-  identifier: true
+**Note!** if you have sources defined in your project's yml, the above will not work. Instead you will need to add the following where your group table is defined in your yaml:
```
Comment (fivetran-sheringuyen) [inline README.md:144]: Thinking about this a bit more -- maybe an example of what sources defined in your project's yml would mean? (and similarly for the source pacakge)

File: packages.yml [hunk 2]
```diff
@@ -1,3 +1,7 @@
 packages:
-- package: fivetran/pendo_source
-  version: [">=0.3.0", "<0.4.0"]
+# - package: fivetran/pendo_source
+#   version: [">=0.4.0", "<0.5.0"] 
+# change before release to above!!!!!
+  - git: https://github.com/fivetran/dbt_pendo_source.git
```
Comment (fivetran-sheringuyen) [inline packages.yml:5]: reminder to update before release

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+# dbt_pendo v0.4.0
+## 🚨 Breaking Changes 🚨:
```
Comment (fivetran-sheringuyen) [inline CHANGELOG.md:2]: would the changes in this package be a breaking change?

**Review by fivetran-sheringuyen — COMMENTED** [review]
Left some in-line comments, let me know if you have questions! I'm going to hold off approving until we can confirm whether this should be a breaking change for the transforms package.

File: README.md (hunk 1 above)
Comment (fivetran-catfritz) [reply README.md]: I have adjusted.

File: CHANGELOG.md (hunk 3 above)
Comment (fivetran-catfritz) [reply CHANGELOG.md:2]: Note for the history--this is a breaking change because of changes in the source.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,10 @@
...
+- This is a breaking change due to changes made in the [source package changelog](https://github.com/fivetran/dbt_pendo_source/blob/main/CHANGELOG.md).
```
Comment (fivetran-sheringuyen) [inline CHANGELOG.md:3]:
```suggestion
- This is a breaking change due to changes made in the source package. ([Source Changelog](https://github.com/fivetran/dbt_pendo_source/blob/main/CHANGELOG.md)). 
```

**Review by fivetran-sheringuyen — APPROVED** [review]
LGTM just some reminders:

- [ ] Update packages.yml before release
- [ ] Close out the relevant issue

File: CHANGELOG.md (hunk 4 above)
Comment (fivetran-catfritz) [reply CHANGELOG.md:3]: Agreed. Updated!

**Pull request merged**